.github/semantic.yml

```diff
@@ -1,2 +0,0 @@
-titleAndCommits: true
-anyCommit: true
```

Comment:
Do we need to add these settings for the GH Action?

Reply:
Not that I am aware of, should work out of the box with the configuration I have in there